### DISCUSSION TITLE
Laptop guidance

### DISCUSSION THREAD

**Discussion by djenson2000** [ppy/osu#20104] [Ideas]
It would be great if Osu! could detect laptop users and suggest to them to use the iGPU, this could be in the same place where the "unable to run exclusive fullscreen message appears"

Thanks to Peppy suggesting I try to use the iGPU, my latency has been much lower, CPU usage has also dropped and this combined with not using the dGPU has lead to decreased temps and almost no stutters (which were common when dGPU was being used)

I'm also curious about DirectX support I see being mentioned. I understand that generally, DirectX plays better with optimus laptops.

**Comment by frenzibyte** [comment]
Direct3D support is coming soon enough, stay tuned.
